dusty/payload.py

```diff
@@ -3,6 +3,14 @@
 
 from .constants import VERSION
 
+def _encode_utf8(args):
```

Comment:
I'll look into the root cause of the issues with `yaml.safe_load` and see if there is something we can do on that side